### PR TITLE
Clean up the osu! World Cup #1 wiki article

### PR DESCRIPTION
**Pull request by Nivalyx**
as it stands a lot of core information on the osu! World Cup #1 wiki article are either incorrect or missing altogether; this pr aims to fill that gap based on all the stuffs i've gathered and researched on this particular topic.

some changes in this pr include : 

- added match results (they were *veeeeeery* scattered throughout the forum thread but thankfully all the required information are still able to be gathered through some sort of workaround in one way or another)
- added tournament schedule (rough estimation, based on the forum timestamp of the referee's posts)
- added missing Finals mappool (the current version of the article wrongly assumed that the Finals and the 3rd Place Playoffs match are using the same mappool)
- added proper tournament staff roster
- reworked the ruleset section to be more in line with modern tournament ruleset writing standards
- added clarification that the tournament was run by various community members with the provision of the osu!team (instead of "being directly run by the osu!team" as currently indicated)
- added proper context for Team Japan's "special" supporter tag prize (they didn't get the supporter tag by winning a "special match" as currently indicated)
- applied bunch of other minor tweaks and improvements

### CODE REVIEW

File: wiki/Tournaments/OWC/1/en.md [hunk 1]
```diff
@@ -10,47 +10,65 @@ tags:
...
-- The OWC started at 2011-02-21.
-- All matches were played from 15:00 to 22:00 (UTC+0), from Thursday to Sunday.
+| Event | Timestamp |
+| --: | :-- |
+| Registration Phase | 2011-01-31/2011-02-21 |
+| Round of 32 | 2011-02-22/2011-02-28 |
+| Round of 16 | 2011-03-01/2011-03-07 |
+| Quarterfinals | 2011-03-08/2011-03-22 |
+| Semifinals | 2011-03-23/2011-03-29 |
+| 3rd Place Playoff & Finals | 2011-03-30/2011-04-11 |
```
Comment (TPGPL) [inline wiki/Tournaments/OWC/1/en.md:24]:
```suggestion
| Finals | 2011-03-30/2011-04-11 |
```

File: wiki/Tournaments/OWC/1/en.md [hunk 2]
```diff
@@ -10,47 +10,65 @@ tags:
...
-| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 1 month of osu!supporter tag |
-| *Special* | 1 month of osu!supporter tag |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 6 months of osu!supporter tag for each team member, unique profile badge |
+| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 1 month of osu!supporter tag for each team member |
+| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 1 month of osu!supporter tag for each team member |
 
 ![](img/badge.jpg "OWC #1 winner badge")
 
-## Organisation
+Aside from the main prizes listed above, the Tournament Management also decided to separately award all members of the ![][flag_JP] Japanese team 1 month of osu!supporter tag each in the wake of the [2011 Tōhoku earthquake](https://en.wikipedia.org/wiki/2011_T%C5%8Dhoku_earthquake_and_tsunami) incident.
```
Comment (TPGPL) [inline wiki/Tournaments/OWC/1/en.md:36]:
```suggestion
Aside from the main prizes listed above, the tournament management has also decided to separately award all members of the ![][flag_JP] Japanese team with 1 month of osu!supporter tag in the wake of the [2011 Tōhoku earthquake](https://en.wikipedia.org/wiki/2011_T%C5%8Dhoku_earthquake_and_tsunami) incident.
```

File: wiki/Tournaments/OWC/1/en.md [hunk 3]
```diff
@@ -10,47 +10,65 @@ tags:
...
-- ![][flag_US] [Derekku](https://osu.ppy.sh/users/91341)
-- ![][flag_JP] [dvorak](https://osu.ppy.sh/users/271359)
-- ![][flag_DE] [Larto](https://osu.ppy.sh/users/12328)
-- ![][flag_IT] [Lunah](https://osu.ppy.sh/users/1227377)
-- ![][flag_AU] [m980](https://osu.ppy.sh/users/3288)
+| Position | Member(s) |
+| :-: | :-- |
+| Host/Manager | ![][flag_IT] [Lunah](https://osu.ppy.sh/users/1227377) |
+| Streamer | ![][flag_JP] [dvorak](https://osu.ppy.sh/users/271359) |
+| Map Selector | ![][flag_IT] [Lunah](https://osu.ppy.sh/users/1227377), ![][flag_AU] [m980](https://osu.ppy.sh/users/3288), ![][flag_DE] [Larto](https://osu.ppy.sh/users/12328), ![][flag_TW] [Alace](https://osu.ppy.sh/users/25993) |
```
Comment (TPGPL) [inline wiki/Tournaments/OWC/1/en.md:46]:
```suggestion
| Mappool selector | ![][flag_IT] [Lunah](https://osu.ppy.sh/users/1227377), ![][flag_AU] [m980](https://osu.ppy.sh/users/3288), ![][flag_DE] [Larto](https://osu.ppy.sh/users/12328), ![][flag_TW] [Alace](https://osu.ppy.sh/users/25993) |
```

**Review by TPGPL — DISMISSED** [review]

**Comment by Nivalyx** [comment]
i decided to keep `Tournament Management` to be written in the way it currently is as the 2020 OWC article has also extensively used `Tournament Management` instead of `tournament management` :

image

image

other suggestions are applied nevertheless, thank you very much as always 🧙

**Review by TicClick — APPROVED** [review]
thank you for your hard work

**Pull request merged**